docs/src/content/docs/introduction/op-concepts.mdx

```diff
@@ -90,12 +90,7 @@ When an `outgoing-payment` is completed against an open and active `incoming-pay
 
 To set up ILP as a payment method in OP, the following are required:
 
-- The <LinkOut href='https://interledger.org/developers/rfcs/ilp-addresses/'>ILP address
-  </LinkOut> of the payee’s ASE: The ILP address is required so that packets representing
-payments routed over the Interledger network will be forwarded to the node owned
-and operated by the intended receiver (i.e. payee's ASE). The `ilpAddress` field
-must be sent by the payee’s ASE in the `methods` object on an `incoming-payment`
-response.
+- The <LinkOut href='https://interledger.org/developers/rfcs/ilp-addresses/'>ILP address </LinkOut> of the payee’s ASE: The ILP address is required so that packets representing payments routed over the Interledger network will be forwarded to the node owned and operated by the intended receiver (i.e. payee's ASE). The `ilpAddress` field must be sent by the payee’s ASE in the `methods` object on an `incoming-payment` response.
```

Comment:
Just fixes a prettier error: https://github.com/interledger/open-payments/actions/runs/8710067273/job/23891288640

Kind of weird - it failed here before I merged in main. But worked locally. Then when I merged main in it started failing locally. So a couple mysteries:

1. how did it get in main in the first place? should have failed in ci on branch
2. why did my pr fail before I had main merged in?